### ISSUE TITLE
Can I use "Competition Categories" at tables instead of styles?

### ISSUE THREAD

**Issue by mlk2b** [geoffhumphrey/brewcompetitiononlineentry#943]
We have a fairly small competition (about 100 entries) and tend to group various styles into Competition Categories. (For example, the Dark Standard category has American Stouts compete with British Porters. The Light Standard category has Czech Pils compete against Maibocks.) In my experience, though, the software only allows one style per table, which isn't feasible in our small competition. Is there a simple way around this so that I can assign multiple styles to one table for judging? 

Here's our site from last year's competition. http://throwdown.starcitybrewers.org/

Thanks for any help you can offer! (I'm pretty new to github and BCOEM so I apologize if this is a newb question!)

**Comment by ercousin** [comment]
What do you mean by "one style per table"? Do you mean style category? Even so, that isn't true, you can group whichever style into whichever table you want using the "Add a Table" step.

Eg.
![image](https://user-images.githubusercontent.com/1551339/42414454-ad726fe2-8203-11e8-9e10-65f0225eb2c8.png)

The installation I'm pasting from already has all tables created, so ignore the red highlighting, but you should see something like this.

![image](https://user-images.githubusercontent.com/1551339/42414470-dd20d1d4-8203-11e8-8e9d-4695b757a2a2.png)

**Comment by mlk2b** [comment]
I might be a tad confused. Thanks for bearing with me. 

I think our biggest issues come with beers like Spice, Herb, and Vegetable beers. Since they're all 30A, the software doesn't like me sending them to different tables. For example, can I send a Vanilla Porter (30A) to a "Dark Specialty" table and a Mango IPA (30A) to a "Light Specialty" table? My understanding is I have to assign the whole category to a single table.

**Comment by ercousin** [comment]
Correct, whole subcategory like 30A must be judged together as far as BCOEM is concerned. The Req Info specified by the entrant is just a text field. There's nothing to stop you from doing this manually, but then you wouldn't be able to report results through BCOEM.

Also, Mango IPA would be 29A Fruit Beer, since it's a fruited version of a classic style.

IMO it's fairer to entrants to separate Specialty by established subcategory than try and interpret entrant intent and create "light specialty" and "dark specialty".
Eg.
Table 5 - American Wild Ale
Table 6 - Fruit Beer
Table 7 - SHV Beer
Table 8 - Other Specialty Beer

**Comment by mlk2b** [comment]
Okay, I think I understand the situation a little better now. Thank you.

My understanding is that it's currently not possible to assign individual entries to tables (you have to assign the whole subcategory). How difficult would it be to develop a version of BCOEM with the option of assigning individual entries to tables, rather than whole subcategories? Thanks again, guys!

**Comment by geoffhumphrey** [comment]
Yes, as it stands in all versions, the entire category of 30A would have to be "lumped" together in any given comp. Logistically, for me personally as a organizer, this is certainly sufficiently granular. However, if you want to lump certain entries together, you can do that too when you define the flights at a given table. Just make sure that you adjust your comp's judging preferences to NOT utilize the queued method. Then, define your table and then define each flight at that table, placing like entries in the same flight.

Judges are certainly used to SHVs being combined and it's the table/flight's Head Judge's responsibility to order the pulling of entries (especially so if queued judging is being utilized); as a result, a natural division will occur based upon what the entrant has declared. A competent HJ would not (and should not) lump dissimilar entries together unless absolutely necessary.

**Comment by mlk2b** [comment]
Okay - thank you, guys.
